composer.json

```diff
@@ -38,26 +38,12 @@
     },
     "autoload": {
         "classmap": [
-            "redaxo/src/addons/backup/lib/",
+            "redaxo/src/addons/*/lib/",
```

Comment:
hier ist jetzt das tests addon auch mit drin, was ich vorher nur in autoload_dev hatte.
wenn wir das hier mit rein haben wollen, sollten wir es aus autoload_dev entfernen.

Wo es letztlich besser ist, bin ich mir nicht sicher.

Insgesamt fand ich hier die explizite Auflistung eigentlich ok. Ist ja nicht so, dass wir ständig neue Addons hier haben.
Mit den globs könnte beim lokalen Ausführen von Composer auch passieren, dass noch Fremdaddons in der classmap landen. Da weiß ich aber auch nicht, ob das eher ein Feature oder ein Problem ist.